docs/modules/ROOT/pages/scope.adoc

```diff
@@ -0,0 +1,26 @@
+= Scope
+
+Secrets often cover some specific aspect of a workload. For example:
+
+* A Kerberos credential may be bound to one node IP
+* An internal TLS certificate's `subjectAlternateName` section must match the `Pod`'s name and service
```

Comment:
Something's off with the formatting here. 

The  's after Pod seems to not trigger the end of the fixed-width formatting.
You might have to use two backticks, I'm not 100% sure.

Reply:
Good point. I thought I was strict with using "`foo` object's" (awkward as that is, too..), but it looks like I missed one case.